backend/src/main/java/com/metamong/mt/domain/admin/controller/AdminController.java

```diff
@@ -0,0 +1,37 @@
+package com.metamong.mt.domain.admin.controller;
+
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.metamong.mt.domain.admin.dto.response.MemberSearchResponseDto;
+import com.metamong.mt.domain.admin.dto.response.ReportedMemberResponseDto;
+import com.metamong.mt.domain.admin.service.AdminService;
+import com.metamong.mt.global.apispec.BaseResponse;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/admin")
+public class AdminController {
+	
+	private final AdminService adminService;
+	
+	@GetMapping("/searchMembers")
```

Comment:
REST 원칙에 맞게 ```-```으로 구분해 주세요
```suggestion
	@GetMapping("/search-members")
```